wazo_load_pilot/plugins/status/http.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2023 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from fastapi import APIRouter
+
+router = APIRouter()
+
+
+@router.get('/status', status_code=200)
+async def get_status():
+    return {"message": "OK"}
```

Comment:
By convention with our other repos, we normally use `Status.ok` or `Status.fail` from the lib-python.status module.  This can be used instead when logic is implemented